src/backend/api/meals.js

```diff
@@ -1,15 +1,97 @@
+const { response, request } = require("express");
 const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
+
+    
+    if('maxPrice' in request.query){
+      const maxPrice = request.query.maxPrice;
```

Comment:
```suggestion
      const maxPrice = Number( request.query.maxPrice);
```

Maxprice must be a number

Reply:
Yes..Thank you :-)